lib/components/footer.jsx

```diff
@@ -109,42 +118,44 @@ export const render = (props) => {
   )
 
   return (
-    <Box
+    <Wrapper
       px={3}
       py={5}
       mt={5}
-      bg="#eee"
       style={{
```

Comment:
This style attribute can just be folded into the styled component you declare above.

Reply:
This is the previous state. Now it uses the theme.